packages/react-badge/src/index.ts

```diff
@@ -1,5 +1,31 @@
-export * from './components/Badge/index';
-export * from './Badge';
-export * from './PresenceBadge';
-export * from './components/CounterBadge/index';
-export * from './CounterBadge';
+export {
+  Badge,
+  badgeClassName,
+  renderBadge_unstable,
+  useBadgeStyles_unstable,
+  useBadge_unstable,
+} from './components/Badge/index';
+export type { BadgeProps, BadgeSlots, BadgeState } from './components/Badge/index';
+export { Badge, badgeClassName, renderBadge_unstable, useBadgeStyles_unstable, useBadge_unstable } from './Badge';
+export type { BadgeProps, BadgeSlots, BadgeState } from './Badge';
+export {
+  PresenceBadge,
+  presenceBadgeClassName,
+  usePresenceBadgeStyles_unstable,
+  usePresenceBadge_unstable,
+} from './PresenceBadge';
+export type { PresenceBadgeProps, PresenceBadgeState, PresenceBadgeStatus } from './PresenceBadge';
```

Comment:
```suggestion
export {
  PresenceBadge,
  presenceBadgeClassName,
  usePresenceBadgeStyles_unstable,
  usePresenceBadge_unstable,
} from './PresenceBadge/index';
export type { PresenceBadgeProps, PresenceBadgeState, PresenceBadgeStatus } from './PresenceBadge/index';
```

We only allow exports from files I believe